data/hai/hai ships.txt

```diff
@@ -733,6 +733,141 @@ ship "Pond Strider" "Pond Strider (Railgun)"
 
 
 
+ship "Sea Scorpion"
+	sprite "ship/hai sea scorpion"
+	thumbnail "thumbnail/hai sea scorpion"
+	attributes
+		category "Medium Warship"
+		"cost" 5450000
+		"shields" 15500
+		"hull" 4900
+		"required crew" 14
+		"bunks" 36
+		"mass" 360
+		"drag" 5.2
+		"heat dissipation" .7
+		"fuel capacity" 500
+		"cargo space" 45
+		"outfit space" 510
+		"weapon capacity" 168
+		"engine capacity" 117
+		weapon
+			"blast radius" 84
+			"shield damage" 840
+			"hull damage" 420
+			"hit force" 1260
+	outfits
+		"Hai Tracker Pod" 4
+		"Chameleon Anti-Missile" 2
+		"Tracker Storage Pod" 4
+		"Hai Tracker" 392
+		"Hai Williwaw Cooling" 2
+		"Boulder Reactor"
+		"Hai Ravine Batteries"
+		"Hai Diamond Regenerator"
+		`"Bondir" Atomic Thruster`
+		`"Bondir" Atomic Steering`
+		"Value Detector"
+		"Hyperdrive"
+		"Pulse Rifle" 15
+
+	engine -28.5 68.5
+		zoom 0.65
+	engine 28.5 68.5
+		zoom 0.65
+	"steering engine" -28.5 68.5
+		zoom 0.35
+		right
+	"steering engine" 28.5 68.5
+		zoom 0.35
+		left
+	"steering engine" 64.5 23.5
+		zoom 0.5
+		angle -90
+		right
+	"steering engine" -64.5 23.5
+		zoom 0.5
+		angle 90
+		left
+	gun -13 -77.5
+	gun 13 -77.5
+	gun -27 -56
+	gun 27 -56
+	turret -35.5 24.5
+	turret 35.5 24.5
+	leak "leak" 40 50
+	leak "flame" 40 30
+	explode "tiny explosion" 24
+	explode "small explosion" 50
+	explode "medium explosion" 30
+	explode "large explosion" 14
+	"final explode" "final explosion medium"
+	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are finally able to mass produce them. The ever growing fleet of Sea Scorpions enables the Unfettered to employ new tactics against the other Hai which alleviate some of their numerical, technological, and logistical disadvantages."
+	description "	One single Sea Scorpion may not hope to do much more than use its mobility to stay clear of the enemy's weapons, using trackers and utilizing its range advantage. A pack of them could easily overwhelm any amount of anti missile put against them. However, where the Sea Scorpion really shine is in small hunting groups, when equipped with pulse weapons. Together, they are able to strike at isolated bigger targets, covering each other and retreating to regenerate their shields if needed, always striking in the enemy's blind spots until they can finally come in for the kill, and plunder."
```

Comment:
Same as the outfits; you don't need to go into exact detail about the specifics on how the Unfettered pilot their ships.
```suggestion
```

Reply:
It adds depht to the game, I like to have a bit of lore hidden in descriptions

Reply:
This isn't lore, this is a basic description of swarm tactics that is not even necessarily exclusive to the Unfettered.

Reply:
Its a story about those ships, how they're meant to work; a living description of sorts
But if removing it is required I'll do it